.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
I don't think your settings should go into your PRs. What is the way to ignore some files in git?

Reply:
Yes, you're right. I added .vscode/settings.json line to the .gitignore file. Thank you.